meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx6ullevk/lmp.cfg

```diff
@@ -55,6 +55,9 @@ CONFIG_FIT_SIGNATURE_STRICT=y
 CONFIG_FIT_VERBOSE=y
 # CONFIG_LEGACY_IMAGE_FORMAT is not set
 CONFIG_PARTITION_UUIDS=y
+CONFIG_FIOVB=y
+# CONFIG_SPL_FIOVB is not set
+CONFIG_CMD_FIOVB=y
```

Comment:
Do we use this for imx6ullevk as it doesn't have eMMC?

Reply:
Sure, thanks. We need to fix this from another side - fiovb command shouldn't be called for modules without eMMC :)